src/Components/WebAssembly/Authentication.Msal/src/Interop/rollup.config.mjs

```diff
@@ -24,7 +24,7 @@ export default ({ environment }) => {
   const baseConfig = {
     output: {
       dir: path.join(__dirname, '/dist', environment === 'development' ? '/Debug' : '/Release'),
-      format: 'cjs',
+      format: 'iife',
```

Comment:
Could you add a comment to indicate why we're using `iife` format here but `es` elsewhere? Same for the other one below.

Reply:
Yeah, these essentially are imported as plain `<script>` tags, while the custom elements one is imported as an ES module. I didn't want to make these ES modules because it would break existing scenarios `<script src="..." />` as they would need to add `type=module`